integrations/maltiverse.py

```diff
@@ -5,8 +5,7 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-"""
-What is Maltiverse?
+"""What is Maltiverse?
```

Comment:
This module docstring looks slightly strange when applying D212 rule since the first line in this case is not a summary but a title.